syntax = "proto3";

import "ltproto/peer2peer/streaming_params.proto";

package ltproto.server;
option java_package = "cn.lanthing.ltproto.server";
option java_outer_classname = "OpenConnectionAckProto";

// 被控端回给服务器
message OpenConnectionAck
{
    enum ErrCode
    {
        Success = 0;
        Invalid = 1;
    }
    ErrCode err_code = 1;
    optional ltproto.peer2peer.StreamingParams streaming_params = 2;
}